integrations/nvidia.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: integration
+name: NVIDIA
+description: Use NVIDIA models with Haystack.
+authors:
+    - name: deepset
+      socials:
+        github: deepset-ai
+        twitter: deepset_ai
+        linkedin: deepset-ai
+pypi: https://pypi.org/project/haystack-ai
+repo: https://github.com/deepset-ai/haystack
```

Comment:
We need to provide links for the integration. Also below in `report_issue` field